### PR TITLE
[DataFactory] Update SDK from swagger.

### PR DESCRIPTION
**Pull request by mdo98**
* Enable Databricks warm pools.

### CODE REVIEW

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @mdo98 , could you
- Add a link to the orginal spec review PR in the description
- Add the `.txt` file generated when you run `generate.ps1`[(details)](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec)
- Update versions, tags, if necessary
- and did you forget to update changelog?

Thanks

**Comment by mdo98** [comment]
@isra-fel @AlexGhiondea @erich-wang Addressed your comments and resolved conflict; please approve and merge asap; we need to release this today.

**Comment by hvermis** [comment]
@isra-fel @AlexGhiondea @erich-wang For ADF please do not ask individual developers to update the version. I do our releases, and it I always end up changing, since the numbers they put are usually wrong. Last version for ADF SDK was 4.1.2, so this one should be 4.1.3.
For reference: https://www.nuget.org/packages/Microsoft.Azure.Management.DataFactory/

**Comment by mdo98** [comment]
@isra-fel @AlexGhiondea @erich-wang By the way the command in the generate.ps1 is incomplete.  I had to use the following command.
powershell -Command "& {Start-AutoRestCodeGeneration -ResourceProvider datafactory/resource-manager -AutoRestVersion latest -ConfigFileTag package-2018-06 -SdkRepoRootPath E:\src\repos\github\mdo98\azure-sdk-for-net\sdk }"

Also, the error message from running the command is also wrong.  It advised users to add the -SdkRepoPath parameter, but the correct parameter is -SdkRepoRootPath.

File: sdk/SDKs/_metadata/datafactory_resource-manager.txt [hunk 1]
```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
```
Comment (erich-wang) [inline sdk/SDKs/_metadata/datafactory_resource-manager.txt]: This .txt should be under `eng\mgmt\mgmtmetadata\`

**Comment by erich-wang** [comment]
> @isra-fel @AlexGhiondea @erich-wang By the way the command in the generate.ps1 is incomplete. I had to use the following command.
> powershell -Command "& {Start-AutoRestCodeGeneration -ResourceProvider datafactory/resource-manager -AutoRestVersion latest -ConfigFileTag package-2018-06 -SdkRepoRootPath E:\src\repos\github\mdo98\azure-sdk-for-net\sdk }"
> 
> Also, the error message from running the command is also wrong. It advised users to add the -SdkRepoPath parameter, but the correct parameter is -SdkRepoRootPath.

@mdo98, have you followed the detail [instruction](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec) to pull latest code and installpsmodules? And use new PS console and navigate to the folder your generate.ps1? You don't need to specify SdkRepoRootPath if you follow the instruction.

**Comment by hvermis** [comment]
@erich-wang, can you please merge this? We need to release our SDK, and we are waiting for this change.

**Review by hvermis — APPROVED** [review]

**Comment by hvermis** [comment]
@isra-fel @AlexGhiondea @erich-wang Can someone please review and merge this ASAP? This is delaying our release schedule.

**Comment by weshaggard** [comment]
@isra-fel @erich-wang looks like the feedback has been addressed and this is blocking their release so I'm assisting @hvermis by merging this PR. If there are additional things that need to be address we can do that in a later PR.

**Pull request merged**